### ISSUE TITLE
[🐛] snyk monitor is crashing in v1.323.1

### ISSUE THREAD

**Issue by michal-dragon** [snyk/snyk#1131]
After the latest release, `snyk monitor` crashes with error
```
Cannot read property 'length' of undefined
```

Running
```
snyk test --severity-threshold=high --org=<< parameters.org >> --all-sub-projects
```
works fine with the following output

```
Organization:      #redacted#
Package manager:   gradle 
Target file:       build.gradle.kts 
Project name:      buildSrc 
Open source:       no 
Project path:      /home/circleci/repo/buildSrc 
Licenses:          enabled
```

- `snyk -v`: v1.323.1
- OS: Linux
- Command run: `snyk monitor --remote-repo-url=${CIRCLE_REPOSITORY_URL} --all-sub-projects --org=<< parameters.org >>`

**Comment by michal-dragon** [comment]
Reverting back to v1.323.0 fixes the issue.

This line seems suspicious:
https://github.com/snyk/snyk/compare/v1.323.0...v1.323.1#diff-2efe993c8fa18a460f73eb44bf5517c8R5

**Comment by michal-dragon** [comment]
Fixed in v1.323.2
